minecraft/authentication.py

```diff
@@ -206,6 +267,15 @@ def sign_out(username, password):
                             {"username": username, "password": password})
 
         if _raise_from_response(res) is None:
```

Comment:
This error was present in the original code (which was apparently never tested), but needs to be fixed anyway: the `signout` and `invalidate` endpoints return a `204 No Content` when successful, but `_raise_from_response` always treats `204` as an error. Therefore, this method always raises an exception after the signout completes normally.

Perhaps `_raise_from_response` should take a keyword argument, e.g. `success_status`, which defaults to `requests.codes.ok` (i.e. `200`), but can be specified as `requests.codes.no_content` (i.e. `204`) in cases like these.